records_mover/records/delimited/hint.py

```diff
@@ -1,29 +1,43 @@
 from typing_inspect import is_literal_type, get_args
```

Comment:
Let's teach the Hint type to have descriptions and be able to generate JSON schema information!